scipy/optimize/tests/test_linprog.py

```diff
@@ -1955,6 +1991,15 @@ class TestLinprogHiGHSSimplexDual(LinprogHiGHSTests):
     method = "highs-ds"
     options = {}
 
+    def test_lad_regression(self):
+        '''The scaled model should be optimal but unscaled model infeasible.'''
+        c, A_ub, b_ub, bnds = l1_regression_prob()
+        res = linprog(c, A_ub=A_ub, b_ub=b_ub, bounds=bnds,
+                      method=self.method, options=self.options)
+        assert_equal(res.status, 4)
```

Comment:
It would be great to check that the solution is essentially valid (e.g. `assert(np.all(res.slack > -1e-6))`). It would be great to check that the simple bounds are met, too.

Reply:
This is a good way check that while `x` was found it is still an infeasible solution -- great idea.  Included in latest push

Reply:
The intent was actually to check that the solution _is_ feasible, albeit with slightly loose tolerances. Along those lines, I would like to check that `x` is within bounds.

Reply:
Got it -- added back in the message test and added upper/lower bounds tests for the solution